tensorflow_addons/layers/tests/max_unpooling_2d_v2_test.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
2021

Reply:
Thank you for pointing this @thaink, I have corrected the year.